tests/CMakeLists.txt

```diff
@@ -1,17 +1,19 @@
 cmake_minimum_required(VERSION 3.0.2)
 
 project(GSLTests CXX)
+enable_testing()
```

Comment:
enable_testing() can be called 2 times now.

Once in this test/CMakeLists.txt and in CMakeLists.txt

I'd rather just move the code in the main cmakelists.txt to this file.

Reply:
[Documentation says][1] that this command should be called in start directory so CTest can operate. If I remove the call in root CMakeLists.txt, `make test` will fail with "No rule" error.

 [1]: https://cmake.org/cmake/help/v3.19/command/enable_testing.html

Reply:
Fair enough. Add a comment instead remarking on why this 2nd enable_testing() call is necessary.